style.css

```diff
@@ -1,18 +1,19 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,100&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;1,100&display=swap");
 
 body {
   margin: 0;
 }
 
 #wrapper {
-  width: 375px;
-  background-image: url('images/Header-bg.png');
+  width: 385px;
+  background-image: url("images/Header-bg.png");
   background-repeat: no-repeat;
   background-size: contain;
   background-position-y: calc(50px);
   background-position-x: -10px;
   margin-left: 24px;
   margin-right: 24px;
+  background-color: #e5e5e5;
```

Comment:
- [ Required ] Please consider applying the right background color to the individual sections of your portfolio, not all of the `I'm Suleiman Welcome To My Portfolio Website` part of your design should not have dark grey background color as currently implemented. Please see the screenshot below and rectify this.

## Figma template Vs Your implmentation
<img src="https://user-images.githubusercontent.com/16392046/205301131-a8f146ab-f449-4b3c-836a-bb4ae0d6b707.png" width="300px" />     <img src="https://user-images.githubusercontent.com/16392046/205301573-b49bc9c7-5b0a-4ead-bdde-dabc1a3d7e22.png" width="300px" />